lib/models/api-key.ts

```diff
@@ -19,6 +19,7 @@ import * as errors from 'balena-errors';
 import type * as BalenaSdk from '..';
 import type { InjectedDependenciesParam, InjectedOptionsParam } from '..';
 import { mergePineOptions } from '../util';
+import { default as applicationModel } from './application';
```

Comment:
Let's change this to a lazy loaded import, so that the CLI doesn't have to load all files unless it's really needed.
```
	const applicationModel = once(() =>
		(require('./application') as typeof import('./application')).default(
			deps,
			opts,
		),
	);
```